driver-core/src/main/java/com/datastax/driver/core/SimpleStatement.java

```diff
@@ -88,6 +91,29 @@ public SimpleStatement(String query, Object... values) {
             throw new IllegalArgumentException("Too many values, the maximum allowed is 65535");
         this.query = query;
         this.values = values;
+        this.namedValues = null;
+    }
+
+    /**
+     * Creates a new {@code SimpleStatement} with the provided query string and named values.
+     * <p/>
+     * This version requires that the query string use named placeholders, for example:
+     * <pre>{@code
+     * new SimpleStatement("SELECT * FROM users WHERE id = :i", ImmutableMap.<String, Object>of("i", 1));}
+     * </pre>
+     * The type of the values will be handled the same way as with anonymous placeholders (see
```

Comment:
I would write "type**s**", and also amend some issues in the javadoc of the other constructor: 

- "Note that the type**s** of the values provided [...] hence the driver cannot know what those value**s** should be"
- The sentence about "The codec to serialize each value will be chosen in the codec registry associated with this session's cluster" seems odd. maybe a remnant from JAVA-1007.